### PR TITLE
Add words about target audience

### PR DESCRIPTION
**Pull request by shardulc**
Comments welcome!

### CODE REVIEW

**Comment by vinitra** [comment]
I like it! I would maybe add in bold the first sentence (that it's for anyone who's applying).  @SolalPirelli, thoughts?

Well worded, @shardulc, thanks!

**Comment by SolalPirelli** [comment]
This goes specifically against what EDIC wanted, as far as I can tell, which is why I didn't originally word the page that way 😅

**Comment by shardulc** [comment]
Oh interesting, I didn't know, @SolalPirelli could you summarize the department's stance about this?

**Comment by SolalPirelli** [comment]
I created #23 based on this.

The idea is that the dept wasn't sure about notions of "underrepresented" groups since those differ between countries, and they wanted to focus more on EPFL students as well (applying to other schools). I think this could be solved with more discussion, but for this year given the timeline.
